tsrc/manifest.py

```diff
@@ -38,3 +42,35 @@ def get_url(self, src):
             if repo_src == src:
                 return repo_url
         raise RepoNotFound(src)
+
+
+def joinurl(prefix, name):
+    res = None
+    if os.path.exists(prefix):
+        res = os.path.join(prefix, name)
+    elif has_scheme(prefix):
+        res = join_with(prefix, name, "/")
+    else:
+        # Assume simple ssh URL like git@host:path, or just host:path
+        res = join_with(prefix, name, ":")
+    if not res.endswith(".git"):
+        res += ".git"
```

Comment:
OK, the fix with clone_prefix has been deemed worse that the get_project_name() url, closing